assets/js/views/app.js

```diff
@@ -0,0 +1,84 @@
+var CommentView = Backbone.View.extend({
+  // a view will create html off the page
+  // what kind of element should I make?
+  tagName: 'li',
+  template: AppTemplates.comments,
+
+  //will be called when the view is first created
+  initialize: function() {
+    this.listenTo(this.model, 'change', this.render);
+  },
+
+  // events - things I want to listen for
+  events: {
+    'change .toggle': 'toggleDone',
+    'click .destroy': 'burnItWithFire'
+  },
+
+  render: function() {
+    var html = this.template(this.model.toJSON());
+    this.$el.html(html);
+    this.$el.toggleClass('completed', this.model.get('done'));
+    return this;
+  },
+
+  // model is all of the underlying data
+
+  toggleDone: function() {
+    this.model.set('done', !this.model.get('done'));
+    this.model.save();
+  },
+
+  burnItWithFire: function() {
+    var _this = this;
+    this.$el.slideUp(function() {
+      _this.model.destroy();
+      _this.remove();
+    });
+  }
+});
+
+var AppView = Backbone.View.extend({
+  template: AppTemplates.app,
```

Comment:
What is your template called? Remember your template file name needs to match what comes after `AppTemplates.`